code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -3304,8 +3313,7 @@
 	desc = "A 'crate', as the marketing called it, of \"The <b>HUNK</b>\" brand chocolate. An early version of the CHUNK box, the HUNK bar was hit by a class action lawsuit and forced to go into bankruptcy and get bought out by the Company when hundreds of customers had their teeth crack from simply attempting to eat the bar."
 	icon_state = "hunk"
 	w_class = SIZE_MEDIUM
-	hitsound = "swing_hit"
-	force = 35 //ILLEGAL LIMIT OF CHOCOLATE
+	force = 35
```

Comment:
I just want to mention here that one hit on the chest from this when the person has no armour on can (and will) cause internal bleeding. (From testing) Not sure if this is the original intention here, just a heads up

Reply:
hunk is admin only.